RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -340,6 +341,13 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
      fitParsVec.push_back(0.);
      fitParsVec.push_back(999.);
      fitParsVec.push_back(false);
+   } else {
```

Comment:
@mariadalfonso 
Thanks for confirming that  tsTOTen == 0 doesn't need to be singled out.
It can indeed be addressed later.